src/dos/dos_code_page.cpp

```diff
@@ -44,6 +45,17 @@ CHECK_NARROWING();
 // - https://www.seasip.info/DOS/CPI/cpi.html
 // - http://kbd-project.org/docs/font-formats/font-formats-3.html
 
+extern void DOS_UpdateCurrentProgramName();
+
+static void notify_code_page_changed()
+{
+	// Re-create various information to match new code page
```

Comment:
minor type: it's usually one word, "recreate".

Reply:
Fixed.